spec/features/paginate_posts_index_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'spec_helper'
+
+describe 'Paginate posts index' do
+  subject { page }
+  describe 'index is paginated' do
+    before do
+      25.times { FactoryGirl.create(:blog_post) }
```

Comment:
Also note that the `FactoryGirl` prefix is not required, you can just do `create(:blog_post)`.